src/FlatFile.FixedLength/Implementation/FixedLengthLineParser.cs

```diff
@@ -25,6 +27,28 @@ public override TEntity ParseLine<TEntity>(string line, TEntity entity)
             return entity;
         }
 
+        private static string GetValueFromLine(string line, int linePosition, IFixedFieldSettingsContainer field)
+        {
+            if (linePosition + field.Length > line.Length)
+            {
+                if ((linePosition + field.Length) - line.Length != field.Length)
+                {
+                    return line.Substring(linePosition);
+                }
+
+                if (field.IsNullable)
+                {
+                    return field.NullValue;
+                }
+
+                throw new IndexOutOfRangeException(
+                    $"The field at {field.Index} with a length of {field.Length} cannot be found on the line because the line is too short. " +
```

Comment:
Could please change `String interpolation` feature of C# 6.0 to old-style `String.Format`?